src/coreComponents/mesh/SurfaceElementSubRegion.hpp

```diff
@@ -215,9 +215,12 @@ class SurfaceElementSubRegion : public ElementSubRegionBase
     constexpr static char const * creationMassString() { return "creationMass"; }
   };
 
-  /// Map between the face elements and the cells
+  /// Map between the surface elements and the cells
   FixedToManyElementRelation m_surfaceElementsToCells;
 
+  /// Unmapped surface elements to nodes map
+  map< localIndex, array1d< globalIndex > > m_unmappedGlobalIndicesInToNodes;
```

Comment:
Unless you need to ordering this should be an `unordered_map`.

Reply:
I have only moved location of this map but I it was originally created by @rrsettgast who knows better than me how it's used. Does it need to be ordered? 
I should have another one in there though which I am pretty sure that can be made unordered.

Reply:
Does not to be sorted as far as I know.